components/redeemdai/DeployProxy.js

```diff
@@ -82,7 +82,6 @@ function DeployProxy({ onPrev, onNext, showErrorMessageAndAllowExiting }) {
         hasAllowance={hasAllowance}
         allowanceLoading={allowanceLoading}
         giveAllowance={giveProxyDaiAllowance}
-        showErrorMessageAndAllowExiting={showErrorMessageAndAllowExiting}
```

Comment:
Well there's two different DeployProxy components, one in redeemdai/DeployProxy.js and one in migratecdp/DeployProxy, but it might be better to have one reusable component that could be used in both flows.

But in any case I realized that I forgot to test how this screen handles errors so will fix that.